### PR TITLE
Adds external storage for assets instructions

### PR DESCRIPTION
**Pull request by BitOne**
**Description**

This PR adds instructions on how to store assets externally.

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: manipulate_pim_data/store_assets_externally.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+When connecting to an external server for storage, reference files and variations are created on the external server. The structure of the folders in which files are stored comes from the PIM.
+Only thumbnails are stored on the PIM server.
+
+Use a SFTP server
+-----------------
+
+.. _Sftp adapter: https://github.com/1up-lab/OneupFlysystemBundle/blob/master/Resources/doc/adapter_sftp.md
+
+Let's assume that you already have a fully functional SFTP server ready to receive files. You will have to setup the adapter in the ``app/config/config.yml`` file.
```
Comment (Doodoune) [inline manipulate_pim_data/store_assets_externally.rst]: In 4.0 the folder app has been removed, so I'm not sure that it's correct

File: manipulate_pim_data/store_assets_externally.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                awss3v3:
+                    client: acme.s3_client
+                    bucket: ~
+                    prefix: ~
+
+.. note::
+
+    Don't forget to add your credentials and the region-id
+
+Then set this service as the value of the client key in the ``app/config/config.yml`` file.
```
Comment (Doodoune) [inline manipulate_pim_data/store_assets_externally.rst]: same than previous comment

File: manipulate_pim_data/store_assets_externally.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+.. _Awss3 adapter: https://github.com/1up-lab/OneupFlysystemBundle/blob/master/Resources/doc/adapter_awss3.md
+
+First of all, you will need to install the following requirements:
+
+``composer --prefer-dist require aws/aws-sdk-php v3.5.0``
+``composer --prefer-dist require league/flysystem-aws-s3-v3 v1.0.6``
+
+In order to use the AwsS3v3 adapter, you first need to create a client object defined as a service.
+This version requires you to use the "v4" of the signature.
```
Comment (jjanvier) [inline manipulate_pim_data/store_assets_externally.rst]: I didn't understand this sentence

Comment (BitOne) [reply manipulate_pim_data/store_assets_externally.rst]: Me neither. It was in the 3.2 documentation. I removed it.

**Pull request merged**